src/stactools/sentinel1/stac.py

```diff
@@ -20,22 +25,36 @@ def create_collection() -> pystac.Collection:
         'constellation': [c.SENTINEL_CONSTELLATION],
         'platform': c.SENTINEL_PLATFORMS,
         'gsd': [c.SENTINEL_RTC_SAR['gsd']],
-        'proj:epsg': c.SENTINEL_RTC_EPSGS,
+        'proj:epsg': c.SENTINEL_RTC_EPSGS
     }
 
     collection = pystac.Collection(
         'sentinel1-rtc-aws',
         description=c.SENTINEL_RTC_DESCRIPTION,
         extent=c.SENTINEL_RTC_EXTENT,
         title='Sentinel-1 RTC CONUS',
-        stac_extensions=['sar', 'sat', 'proj'],
+        stac_extensions=[
+            SarExtension.get_schema_uri(),
+            SatExtension.get_schema_uri(),
+            ProjectionExtension.get_schema_uri(),
+            RasterExtension.get_schema_uri(),
+            # Can use pystac.extensions once implemented
+            # 'https://stac-extensions.github.io/processing/v1.0.0/schema.json',
+            'https://stac-extensions.github.io/mgrs/v1.0.0/schema.json',
+        ],
         keywords=[
             'backscatter', 'radiometry', 'sentinel', 'copernicus', 'esa', 'sar'
         ],
         providers=[c.SENTINEL_PROVIDER, c.SENTINEL_RTC_PROVIDER],
-        summaries=summary_dict,
+        summaries=Summaries(summary_dict),
     )
 
+    # Fails validation i think b/c 'about' is not a recognized reltype
```

Comment:
I don't see an enum in the `reltype` definition in https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/item.json. Can you post the validation failure?

Reply:
not pursuing this since i can put this information under 'providers' with the processing stac extension